### ISSUE TITLE
Single strand DNA or RNA nodes?

### ISSUE THREAD

**Issue by pgb3** [BradyAJohnston/MolecularNodes#187]
With Molecular Nodes 2.4.3 I cannot find the single strand DNA node. In the double strand node there is no option to use only one of the strands as you show in your tutorials. Is this intentional or a bug?

intel Mac OS 11.6
Blender 2.4 and 2.5

**Comment by BradyAJohnston** [comment]
Hi mate. You can delete one of the strands by using 'delete geometry' and using a selection of `chain_id == 1` or `chain_id == 2` currently. 

I should bring back the single strands though, as it seems people would like the convenience of it.

**Comment by pgb3** [comment]
Thank you very much. I could not find a way to select chain_id==1. I am still a beginner using nodes. What I did that worked is to duplicate the Mol_DNA_double_helix node and rename it so that I did not modify the original one. To the duplicate I deleted one of the Mol_utils_helix node inside. That produced a single strand helix. That worked for what I wanted to do. A dedicated single strand node would be great. Thank you so much for creating this amazing add-on.
